src/pybind/mgr/dashboard/services/tcmu_service.py

```diff
@@ -61,7 +61,7 @@ def get_iscsi_info():
                 perf_key = "{}lock_acquired_time".format(perf_key_prefix)
                 lock_acquired_time = (mgr.get_counter(
                     'tcmu-runner', service_id, perf_key)[perf_key]
-                    or [[0, 0]])[-1][1] / 1000000000
+                                      or [[0, 0]])[-1][1] / 1000000000
```

Comment:
I'm a bit confused which format is correct now, because I think it reverts what has just been introduced: https://github.com/ceph/ceph/commit/0c3f9480a5673ab6eb3a0e60b0fae9665dbfe700#diff-bcad615225e18a074734d863987a2282R64

I had the same issue on my local environment and figured out it was related to outdated installed packages: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/requirements-lint.txt

Reply:
@s0nea I re-created my local tox env. and found that both changes are correct, so this change is undone.
Thanks!